cluster/log-dump/log-dump.sh

```diff
@@ -136,25 +137,25 @@ function copy-logs-from-node() {
 function save-logs() {
     local -r node_name="${1}"
     local -r dir="${2}"
-    local files="${3}"
+    local save_files="${3}"
     local opt_systemd_services="${4:-""}"
     local on_master="${5:-"false"}"
 
     if [[ -n "${use_custom_instance_list}" ]]; then
       if [[ -n "${LOG_DUMP_SAVE_LOGS:-}" ]]; then
-        files="${files} ${LOG_DUMP_SAVE_LOGS:-}"
+        save_files="${save_files} ${LOG_DUMP_SAVE_LOGS:-}"
       fi
     else
       case "${KUBERNETES_PROVIDER}" in
         gce|gke|kubernetes-anywhere)
-          files="${files} ${gce_logfiles}"
+          save_files="${save_files} ${gce_logfiles}"
           ;;
         aws)
-          files="${files} ${aws_logfiles}"
+          save_files="${save_files} ${aws_logfiles}"
           ;;
       esac
     fi
-    local -r services=( ${systemd_services} ${opt_systemd_services} ${LOG_DUMP_SAVE_SERVICES:-} )
+    read -r -a services <<< "${systemd_services} ${opt_systemd_services} ${LOG_DUMP_SAVE_SERVICES:-}"
```

Comment:
shellcheck detects SC2206.
```
In ./cluster/log-dump/log-dump.sh line 157:
    local -r services=( ${systemd_services} ${opt_systemd_services} ${LOG_DUMP_SAVE_SERVICES:-} )
                        ^-----------------^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
                                            ^---------------------^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
                                                                    ^-------------------------^ SC2206: Quote to prevent word splitting, or split robustly with mapfile or read -a.
```